### DISCUSSION TITLE
1.189.0 broke copilot completions

### DISCUSSION THREAD

**Discussion by cmhac** [community/community#123301] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

VSCode info: 

Version: 1.89.1 (user setup)
Commit: dc96b837cf6bb4af9cd736aa3af08cf8279f7685
Date: 2024-05-07T05:13:33.891Z
Electron: 28.2.8
ElectronBuildId: 27744544
Chromium: 120.0.6099.291
Node.js: 18.18.2
V8: 12.0.267.19-electron.0
OS: Windows_NT x64 10.0.22631

Looks like 1.189.0 was released about an hour ago. It now has completely broken completions for me.

When trying to run any commands from the extension, I get these error messages: 

![image](https://github.com/community/community/assets/49451910/482595c8-decf-46da-9d55-e7d7d45afe92)

![image](https://github.com/community/community/assets/49451910/c076040a-1f41-4ec1-8c4f-3bdfc2d096f0)

Downgrading to 1.188.0 fixed it.

**Comment by lancewilhelm** [comment]
Seeing the same thing and downgrading worked for me too. To me it appears native to WSL. Is that what you are on?

**Reply by cmhac** [reply]
Yeah looks like on WSL and also in codespaces, but on windows 1.189.0 is working fine

**Reply by gitgud5000** [reply]
Same, working in Ubuntu Remote SSH. Downgrading to 1.186 worked forme. 1.189 works fine in windows.

**Comment by astrajoan** [comment]
Seeing this as well, and downgrading from 1.189 to 1.188 fixed the issue. Local machine is macOS 14.4 and remote SSH server is Ubuntu 22.04.

**Comment by sromexs** [comment]
I'm on WSL and i see this problem too and downgrading fixes the problem

**Comment by hsenag** [comment]
Hi all, sorry about this. I can reproduce the problem, we'll figure it out and get a fix out as soon as possible.

**Comment by hsenag** [comment, marked as answer]
1.190.0 is out now and should fix the problem. Apologies again, and thanks for all the reports.

**Reply by sromexs** [reply]
thanks

**Comment by RezaShamsipour** [comment]
@hsenag
i also get that error after 4 days . i updated this to 1.190.0
